3rdparty/ndsrvp/src/integral.cpp

```diff
@@ -3,6 +3,8 @@
 // of this distribution and at http://opencv.org/license.html.	
 
 #include "ndsrvp_hal.hpp"
+#include "opencv2/imgproc/hal/interface.h"
+#include "opencv2/core.hpp"
```

Comment:
What is `opencv2/core.hpp` for? By design HAL should not use anything from the main library except for `*/hal/interface.h` headers.

Reply:
Inclusion of `opencv2/core.hpp` has been removed. Related functions are moved to `cvutils.hpp` and `cvutils.cpp`.